### DISCUSSION TITLE
Does the algorithm apply histogram equalization by default?

### DISCUSSION THREAD

**Discussion by amdenisaionascu** [ultralytics/yolov5#8089] [Q&A] [answered]
I have found the hist_equalize function in the augmentations.py but it seemed to me that it is not used. 
Is histogram equalization applied by default? And if so, is it applied only as augmentation or as preprocessing to both train/validation/test sets?

**Comment by glenn-jocher** [comment, marked as answer]
not used

**Reply by amdenisaionascu** [reply]
Thank you!
